### PR TITLE
Release-Prod-05092022

### PR DESCRIPTION
**Pull request by xiangzhe0812**

### CODE REVIEW

File: src/services/vendors/EasyCrypto.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            .catch(error => {
+                if (error.response) {
+                    this.logger.error(error.response.data);
+                    throw new Error(error.response.data);
+                }
+                this.logger.error(error.message);
+                throw error;
+            });
+        const userId = response.data;
+        return { userId };
```
Comment (brettvniekerk) [inline src/services/vendors/EasyCrypto.ts:35]: Why not just return `userId`?

File: src/services/vendors/GPIB.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            .catch(error => {
+                if (error.response) {
+                    this.logger.error(error.response.data);
+                    throw new Error(error.response.data);
+                }
+                this.logger.error(error.message);
+                throw error;
+            });
+        const userId = response.data;
+        return { userId };
```
Comment (brettvniekerk) [inline src/services/vendors/GPIB.ts:46]: Why not just return userId?

File: src/services/vendors/CoinStash.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            .catch(error => {
+                if (error.response) {
+                    this.logger.error(error.response.data);
+                    throw new Error(error.response.data);
+                }
+                this.logger.error(error.message);
+                throw error;
+            });
+        const userId = response.data;
+        return { userId };
```
Comment (brettvniekerk) [inline src/services/vendors/CoinStash.ts:38]: Why not just return userId?

File: src/services/vendors/DigitalSurge.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            .catch(error => {
+                if (error.response) {
+                    this.logger.error(error.response.data.message);
+                    throw new Error(error.response.data.message);
+                }
+                this.logger.error(error.message);
+                throw error;
+            });
+        const { token, password } = response.data;
+        return { userId: token, password: password };
```
Comment (brettvniekerk) [inline src/services/vendors/DigitalSurge.ts:49]: `return { userId: token, password }`

File: src/utils/wallet.spec.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { expect } from "chai";
+import { signMessage, verifyMessage } from "./wallet";
+
+const mockConfigForWallet1: any = {
+    get: () =>
+        "afdfd9c3d2095ef696594f6cedcae59e72dcd697e2a7521b1578140422a4f890"
```
Comment (brettvniekerk) [inline src/utils/wallet.spec.ts:7]: magic string? maybe better as .env variable?

File: src/utils/wallet.spec.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+const mockConfigForWallet2: any = {
+    get: () =>
+        "8da4ef21b864d2cc526dbdb2a120bd2874c36c9d0a1fb7f8c63d7f7a8b41de8f"
```
Comment (brettvniekerk) [inline src/utils/wallet.spec.ts:12]: same here

**Review by bitcoinbrisbane — APPROVED** [review]

File: src/utils/wallet.spec.ts (hunk 5 above)
Comment (brettvniekerk) [inline src/utils/wallet.spec.ts:7]: Could be done as `const TEST_PRIV_KEY_ONE = etc...` and then passed in here but not a big deal

**Review by brettvniekerk — CHANGES_REQUESTED** [review]
Minor things

**Pull request merged**